test/integration/ipamd/warm_target_test.go

```diff
@@ -34,6 +34,22 @@ var _ = Describe("test warm target variables", func() {
 		var maxENI int
 
 		JustBeforeEach(func() {
+
+			By("Verify number of enis and ips before updating the parameters")
```

Comment:
Since this is called in multiple places, let's turn it into a helper function. Also, I think 2 mins may not be enough. Let's start with 5 mins

Reply:
I was expecting few more conditions to check for different tests. I can extract it into a function for now

Reply:
Also, I was under the impression if recycle duration is 30s then it should ideally finish the cleanup in about a minute right for the worst case ?  That's why I had made it to 2 mins just to leave some headroom. I can surely increase it to 5 mins if it's necessary.

Reply:
Ah, I'm not sure what the duration is. If it is 30s, then this sounds reasonable. We can start with 2m knowing that if it takes longer, it is something we want to investigate.

Reply:
Increased it to 3 mins just to be on safer side and not have into flaky tests.